traffic_ops/testing/api/v14/user_test.go

```diff
@@ -42,12 +45,59 @@ func CreateTestUsers(t *testing.T) {
 
 		resp, _, err := TOSession.CreateUser(&user)
 		if err != nil {
-			t.Fatalf("could not CREATE user: %v\n", err)
+			t.Errorf("could not CREATE user: %v\n", err)
 		}
 		log.Debugln("Response: ", resp.Alerts)
 	}
 }
 
+func RolenameCapitalizationTest(t *testing.T) {
+
+	var err error
+	var jsonBytes []byte
+
+	user := tc.User{}
+	blob := []byte(`
+	{
+		"username": "test_user",
+		"email": "cooldude6@comcast.net",
```

Comment:
probably shouldn't use a real domain here, that could potentially be a real email address. Better to use example.com

Reply:
I'll change it..

but comcast specifically has a naming convention that wouldn't have 'cooldude6', nor is it used for anything.

Reply:
Not used for anything today. The naming standard could change in the future.

Reply:
>Same goes for example IP addresses/CIDRs, generally you're supposed to use the designated "example" ranges that are reserved just for that

`192.0.2.0/24` and `2001:DB8::/32` are one of each.

https://tools.ietf.org/html/rfc5737
https://tools.ietf.org/html/rfc3849
https://tools.ietf.org/html/rfc2606